index.html

```diff
@@ -4053,6 +4074,34 @@ <h3>Add HTML Defaults</h3>
 					</ol>
 				</section>
 			</section>
+
+			<section id="processing-pub-resources">
+				<h3>Obtain a List of Publication Resources</h3>
```

Comment:
This section (and it's siblings) define API like activities...but no API names. Their "return ____" statements don't say what they're returning into. Perhaps some text could be added to the parent section describing how all of these are populating a single object/representation?

Reply:
We're not defining APIs, though, only the common steps for a user agent to internally process the information. That's generally how infra is used to define these algorithms.

For some other random samples, see:
- https://dom.spec.whatwg.org/#ordered-sets
- https://encoding.spec.whatwg.org/#concept-encoding-run
- https://fetch.spec.whatwg.org/#collect-an-http-quoted-string
- https://url.spec.whatwg.org/#concept-url-parser